packages/prompts/README.md

```diff
@@ -156,3 +156,66 @@ const group = await p.group(
 
 console.log(group.name, group.age, group.color);
 ```
+
+### Workflow
+
+Just like `group`, but on builder way, so you can choose which one fits better.
+
+```js
+import * as p from '@clack/prompts';
+
+const results = await p
+  .workflow()
```

Comment:
@Mist3rBru You think it would be nice to have an option within workflow that `auto` ran the `run()` function to give user more flexibility?